tasks/mynt.ts

```diff
@@ -965,14 +981,115 @@ task(
         [basketManagerProxy.address, newBasketManagerImpl.address]
       );
 
-      const sipArgs: ISipArgument = {
+      const sipArgs: ISipArgument["args"] = {
         targets: [basketManagerProxy.address],
         values: [0],
         signatures: [signatureUpgrade],
         data: [dataUpgrade],
         description: "Upgrade fees vault contract",
       };
-
-      createSIP(hre, sipArgs);
+      logger.warn(">>> CREATE A SIP WITH THIS ARGS: <<<");
+      logger.info(sipArgs);
+      logger.warn("====================================");
     }
   });
+
+task("upgrade:mocIntegration", "Upgrade implementation of mocIntegration contract")
+.addOptionalParam("isMultisig", "flag if transaction needs to be intiated from the multisig contract")
+.addOptionalParam("isSIP", "flag if transaction needs to be initiated from the SIP")
+.setAction(async ({ isMultisig, isSIP }, hre) => {
+  helpers.injectHre(hre);
```

Comment:
helpers are not defined - that's what linter says

Reply:
updated in this [commit](https://github.com/DistributedCollective/mynt/pull/17/commits/57307ba37bc62d3f003c1b59b86350579a01d092)